_opensearch/snapshots/searchable_snapshot.md

```diff
@@ -0,0 +1,119 @@
+
+The flag is toggled using a new jvm parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+- Option 1: Update config/jvm.options by adding the following line:
+
+    ```json
+    -Dopensearch.experimental.feature.searchable_snapshot.enabled=true
+    ```
+
+- Option 2: Use the `OPENSEARCH_JAVA_OPTS` environment variable:
+
+    ```json
+    export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.searchable_snapshot.enabled=true"
+    ```
+- Option 3: For developers using Gradle, update run.gradle by adding the following lines:
+
+    ```json
+    testClusters {
+      runTask {
+        testDistribution = 'archive'
+        if (numZones > 1) numberOfZones = numZones
+        if (numNodes > 1) numberOfNodes = numNodes
+        systemProperty 'opensearch.experimental.feature.searchable_snapshot.enabled', 'true'
+      }
+    }
+    ```
+
+- Finally, create a node in your opensearch.yml file and define the node role as `search`:
+
+    ```bash
+    node.name: snapshots-node
+    node.roles: [ search ]
+    ```
+
+### Enable with Docker containers
+
+If you're running Docker, add the following line to docker-compose.yml underneath the `opensearch-node` and `environment` section:
+
+```json
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.searchable_snapshot.enabled=true" # Enables searchable snapshot
+```
+
+To create a node with the `search` node roll, add the line `- node.roles: [ search ]` to your docker-compose.yml file:
+
+```bash
+version: '3'
+services:
+  opensearch-node1:
+    image: opensearchproject/opensearch:2.4.0
+    container_name: opensearch-node1
+    environment:
+      - cluster.name=opensearch-cluster
+      - node.name=opensearch-node1
+      - node.roles: [ search ]
+```
+
+## Create a searchable snapshot index
+
+Creating a searchable snapshot index is done by specifying the `remote_snapshot` storage type using the [restore snapshots API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#restore-snapshots).
+
+Request field | Description
+:--- | :---
+`storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the `search` node role in order to restore a snapshot using the type `remote_snapshot`. <br /><br > Defaults to `local`.
+
+## Listing indexes
+
+To determine if an index is a searchable snapshot index, look for a store type with the value of `remote_snapshot`:
+
+```
+GET /my-index/_settings?pretty
+```
+
+```json
+{
+  "my-index": {
+    "settings": {
+      "index": {
+        "store": {
+          "type": "remote_snapshot"
+        }
+      }
+    }
+  }
+}
+```
+
+## Potential use cases
+
```

Comment:
We need an intro sentence here. Something like "The following are potential use cases for the searchable snapshots feature:" is fine. Also, the two bulleted items would need to start with something other than "Users", as users are not use cases. Per the comments below, I would consider removing this section or folding it into the introduction.